File: src/primus/pawel/Exercise_193.java
package primus.pawel;

// Find Screen Size
// 7kyu
// https://www.codewars.com/kata/5bbd279c8f8bbd5ee500000f/train/java
// 2021-08-02 T:23:00:01
class Exercise_193 {

    public static String findScreenHeight(int width, String ratio) {
        String[] arr = ratio.split(":");

        double height = Double.parseDouble(arr[1]) / Double.parseDouble(arr[0]) * width;
        return width + "x" + (int)height;
    }


    public static void main(String[] args) {
        System.out.println(findScreenHeight(3840, "32:9"));
    }
}

File: src/JS/Exercise_193.js
// Pairs of integers from m to n
// 7kyu
// https://www.codewars.com/kata/588e2a1ad1140d31cb00008c/train/javascript
// 2022-08-10 T:07:47:42

function generatePairs(m, n) {
    var resultArray = [];
    for (let i = m; i <= n; i++) {
        for (let j = i; j <= n; j++) {
            resultArray.push([i, j]);
        }
    }
    return resultArray;
}

console.log(generatePairs(2, 4));